src/app/api/payments/route.ts

```
@@ -0,0 +1,165 @@
+        // The service name is not available, but we still think it's a VPN
+        console.error("VPN detected", decision);
+        return NextResponse.json(
+          { success: false, message: "Forbidden" },
+          { status: 403 },
+        );
+      }
+    }
+
+    // Only allow users from the US and UK
+    if (
+      decision.ip.hasCountry() &&
```

ESLint

[error] '&&' should be placed at the beginning of the line.

(style/operator-linebreak)